### ISSUE TITLE
Wrong lr scheduling curve caused by using timm

### ISSUE THREAD

**Issue by mobulan** [microsoft/Swin-Transformer#277]
Hello, I found a problem with the learning rate scheduling in the code. After warm up is set, the learning rate is suddenly reduced by a part when switching back to the normal learning rate. It is more obvious when the epoch value of warm up is larger, as shown in the following figure.
![image](https://user-images.githubusercontent.com/41856770/193806600-6eaabce6-864b-41a9-af9e-34c3b6efe811.png)

I found that this problem is caused by not opening the warm up perfix in timm's lr scheduler, and setting it to True can solve this problem. Because the warm up perfix is not turned on, the cosine annealing curve starts to decay at the 0th step, and then it is abnormally spliced with the learning rate rising to the base lr during warm up epochs.
Suggest：
lr_scheduler.py line33 Add warmup_perfix = True

**Comment by zeliu98** [comment]
Hi @mobulan, thanks for pointing out this, we have fixed it: https://github.com/microsoft/Swin-Transformer/pull/278
